README.md

```diff
@@ -100,137 +45,118 @@ For more information on the importance of a professional README for your reposit
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+- Transactions: Users can create transactions, which are associated with a category. The category can be anything, such as "Food," "Entertainment," or "Transportation."
 
-## 🚀 Live Demo <a name="live-demo"></a>
+- Categories: Users can create and manage categories. This allows them to track their expenses more granularly.
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] As requested by the previous CR please 🙏🏽 deploy your app and add the link to the README file for professional documentation 👇🏽 

![image](https://github.com/Ridwanullahi-code/BudgetApp/assets/58642949/f3030b2f-6caf-4872-b755-a609b2895e14)